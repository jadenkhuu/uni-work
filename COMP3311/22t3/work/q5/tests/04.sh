python3 ./q5 Flemington 2019-08-10
