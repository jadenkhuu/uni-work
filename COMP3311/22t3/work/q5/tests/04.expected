Race meeting at Flemington on 2019-08-10

Race 1, prize pool $3000, run over 2100m
Pony of my Ownie ridden by Rajiv Maragh wins $2100
Jersey Shorse ridden by Kieren Fallon wins $600
Isinglass ridden by Bob Skelton wins $300

Race 2, prize pool $6000, run over 1000m
Justine Thyme ridden by Willie Robinson wins $4200
Horsey Kiss ridden by Rosemary Homeister Jr. wins $1200
Hoof Hearted ridden by William Buick wins $600

Aurie's Star Handicap, prize pool $160000, run over 1200m
Bored Bronco ridden by Paddy Brennan wins $112000
Enterprise ridden by Jean Cruguet wins $32000
Singapore ridden by Tommy Carberry wins $16000
